Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.cs

```diff
@@ -352,9 +354,53 @@ public async Task LoadImageFromFile(StorageFile imageFile)
         /// Gets the cropped image.
         /// </summary>
         /// <returns>The cropped image.</returns>
-        public async Task<WriteableBitmap> GetCroppedBitmapAsync()
+        public async Task<WriteableBitmap> GetCroppedImageAsync()
```

Comment:
Remove this which can be accomplished easily enough through the new SaveAsync method

Reply:
I removed it.